cvpy/annotation/cvat/CVATProject.py

```diff
@@ -367,16 +367,35 @@ def resume(project_name: str, cas_connection: CAS, caslib: str, relative_path: s
         cas_connection:
             Specifies the CAS connection in which the project will be resumed.
         caslib:
-            Specifies the caslib under which CAS tables were saved.
+            Specifies the caslib under which CAS tables were saved. If this is not specified, the active caslib will be used.
         relative_path:
-            Specifies the path relative to caslib where project was saved.
+            Specifies the path relative to caslib where project was saved. If this is not specified, the active caslib path will be used.
 
         Returns
         -------
         project:
             A CVATProject object with all of the properties set from the specified JSON string.
         """
 
+        # Check which parameters were specified
+        if caslib == None and relative_path == None:
+            # Set caslib to the active caslib
+            caslibinfo = cas_connection.caslibinfo()['CASLibInfo']
+            caslib = caslibinfo[caslibinfo.Active == 1].Name.values[0]
+            # Set relative path accordingly
+            relative_path = project_name
+
+        elif caslib == None and relative_path != None:
+            # Set caslib to the active caslib
+            caslibinfo = cas_connection.caslibinfo()['CASLibInfo']
+            caslib = caslibinfo[caslibinfo.Active == 1].Name.values[0]
+            # Add the project name to the relative path
+            relative_path = f"{relative_path}/{project_name}"
+            
+        elif relative_path == None and caslib != None:
+            # Set relative path accordingly
+            relative_path = project_name
+
```

Comment:
Lines 383-384 and 390-391 are identical. I will suggest refactoring the code to avoid such repetition and to make it more maintainable. Line 380-397 can be rewritten as below. 

```
if caslib == None:
    caslibinfo = cas_connection.caslibinfo()['CASLibInfo']
    caslib = caslibinfo[caslibinfo.Active == 1].Name.values[0]

if relative_path == None:
    relative_path = project_name
else:
    relative_path = f'{relative_path}/{project_name}'
```